public/app/features/alerting/unified/components/receivers/TemplateForm.tsx

```diff
@@ -378,8 +388,8 @@ export function TemplatePreview({
   useEffect(() => onPreview(), [onPreview]);
 
   return (
-    <Stack direction="row" alignItems="center" gap={2}>
-      <Stack direction="column">
+    <div style={{ width: `${width}px` }}>
```

Comment:
Why do we need two `div` containers here? Can't we achieve the same using one `div`?

Reply:
good catch! solved and pushed! @konrad147